pkg/graveler/committed/merge_test.go

```diff
@@ -626,29 +687,20 @@ func Test_merge(t *testing.T) {
 			}},
 		},
 		"empty source and base": {
-			baseRange:   []testRange{},
-			sourceRange: []testRange{},
-			destRange: []testRange{
+			baseRange:   newTestMetaRange([]testRange{}),
+			sourceRange: newTestMetaRange([]testRange{}),
+			destRange: newTestMetaRange([]testRange{
 				{rng: committed.Range{ID: "dest:k1-k2", MinKey: committed.Key("k1"), MaxKey: committed.Key("k2"), Count: 2, EstimatedSize: 1234}},
 				{rng: committed.Range{ID: "base:k3-k6", MinKey: committed.Key("k3"), MaxKey: committed.Key("k6"), Count: 2, EstimatedSize: 1234}},
-			},
+			}),
 			expectedResult: []testRunResult{{
 				mergeStrategies: []graveler.MergeStrategy{graveler.MergeStrategyNone, graveler.MergeStrategyDest, graveler.MergeStrategySource},
-				expectedActions: []writeAction{
-					{
-						action: actionTypeWriteRange,
-						rng:    committed.Range{ID: "dest:k1-k2", MinKey: committed.Key("k1"), MaxKey: committed.Key("k2"), Count: 2, EstimatedSize: 1234},
-					},
-					{
-						action: actionTypeWriteRange,
-						rng:    committed.Range{ID: "base:k3-k6", MinKey: committed.Key("k3"), MaxKey: committed.Key("k6"), Count: 2, EstimatedSize: 1234},
-					},
-				},
-				expectedErr: nil,
+				expectedActions: []writeAction{},
```

Comment:
Concretely, I believe @nopcoder is correct and I do not like this expected result -- I believe the old one was correct.  So I would appreciate an explanation of this change.

Reply:
Explained by @guy-har .  THANKS!